content/guides/getting-started-with-hugo.md

```diff
@@ -0,0 +1,178 @@
+
+### Static:
+
+This is, of course, where your static elements will exist. These elements may include your javascript or css files.
+
+### config.toml:
+This file contains the settings of your website, such as its URL, what language it contains, and the title of the website.
+
+### Themes:
+If graphic design isn’t one of your strengths, Hugo being an open-source project means that there are countless persons releasing their own themes and layouts for you to use! You can browse these themes [here](https://themes.gohugo.io "Hugo Themes"). Installing a theme is just as simple as downloading a folder to your machine, because that is exactly what you are doing! You may notice that every theme which peaks your interest will also exist on GitHub. You can use whatever method you prefer to retrieve the code. In this document, I will demonstrate a couple ways to do this. Once you find a theme that suits your needs or ‘aesthetic’, simply click on the theme. You may see two or three buttons, ‘Download’, ‘Demo’, ‘Homepage’. If you are confident that this is the theme you want, click ‘Download’ and it will take you to the GitHub repository where the theme exists. Click on the green ‘Code’ button and this will give you a few options on how to retrieve the code. 
+
+ ![Git-Clone-Theme](images/getting-started/GitHub-Code-Options.png "download theme options")
+
+For this example, I have found a theme that interests me titled ‘Dream’. For users who have Git, you can copy the https link and navigate into the ‘themes’ folder of your project to git clone it. Refer to the screenshot below (I am using Git Bash):
+
+ ![Git-Clone-Theme](images/getting-started/Git-Clone-Theme.png "git bash theme clone")
+
+When you clone the repository, it will automatically create a folder for you. However, if you are not sure how to use Git, you can simply download the Zip file onto your machine. Before installing this zip folder, be sure to create a folder within your ‘themes’ folder with the appropriate title of the theme you are downloading. After downloading the zip folder, you should extract the contents into the folder that you just created inside of your ‘themes’ folder. You can see in the screenshot below, that the folder ‘hugo-theme-dream’ exists within the ‘themes’ folder. 
+
+ ![Theme-File-Location](images/getting-started/Theme-File-Location.png "themes folder")
+
+After successfully implementing the above steps, navigate into your config.toml file and add the line of code below everything else that exists in the file:
+
+```
+theme = “folder_name_here”
+```
+
+So, in my case, I would type the following:
+
+```
+theme = “hugo-theme-dream”
+```
+
+After adding this line, save the document and to make sure that you have properly called this theme into your site use the following command in your command prompt/terminal to start up your Hugo local server:
+
+```dart
+hugo server -D
+// The -D will launch all of the draft pages that
```

Comment:
comments probably shouldn't be in the code block, especially now that GitHub has that copy button that means they're going to get pasted into somebody's shell.